java/src/main/native/OrtJniUtil.h

```diff
@@ -11,6 +11,30 @@
 extern "C" {
 #endif
 
+inline jsize safecast_size_t_to_jsize(size_t v) {
+#ifdef _DEBUG
```

Comment:
should be #ifndef NDEBUG.  NDEBUG is required to be set up correctly to make assert work.